test/integration/daemonset/daemonset_test.go

```diff
@@ -17,246 +17,14 @@ limitations under the License.
 package daemonset
 
 import (
-	"fmt"
-	"net/http/httptest"
 	"testing"
-	"time"
 
-	apps "k8s.io/api/apps/v1"
 	"k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/resource"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/intstr"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/informers"
-	clientset "k8s.io/client-go/kubernetes"
-	appstyped "k8s.io/client-go/kubernetes/typed/apps/v1"
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
 	corev1typed "k8s.io/client-go/kubernetes/typed/core/v1"
-	restclient "k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/cache"
-	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
-	"k8s.io/kubernetes/pkg/controller/daemon"
-	"k8s.io/kubernetes/pkg/util/metrics"
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
-func setup(t *testing.T) (*httptest.Server, framework.CloseFunc, *daemon.DaemonSetsController, informers.SharedInformerFactory, clientset.Interface) {
```

Comment:
Shouldn't setup be back here and not in the util class? We should look to be consistent with the other test files.

Reply:
Perhaps we should change other test files. It may be better to only have tests in `daemonset_test.go` file. For all helper functions, there should be in the `util.go` file.

I do not have any strong opinion though.

Reply:
As sync by person, `setup()` is tightly coupled with the integration tests, so it makes more sense to include it in the `daemonset_test.go` (just like other controllers' integration test files).